eng/scripts/Show-FailureLogs.ps1

```diff
@@ -15,14 +16,33 @@ if (!$filteredLogs) {
     exit 0
 }
 
-foreach ($logFile in $filteredLogs)
+$filteredLogs = $filteredLogs | foreach {$_.FullName}
```

Comment:
```suggestion
$filteredLogs = @($filteredLogs.FullName)
```

This should be cast to an array type, otherwise powershell will try to make it the element type if length is 1, which can cause issues downstream.

Reply:
Thank you!